README.md

```diff
@@ -1,4 +1,18 @@
-## Introduction
+## What is this
 
-A play-cljs game in which ... well, that part is up to you. Do `boot run` to develop and `boot build` to compile a release version.
+A turn-based tactical game... in space!
 
+## How do I work on it
+
+### Development
+
+To run a development server with file watching, run `boot run`.
+
+You can get a repl for Clojure files (.clj, .clj) with `boot repl`. Run it from
+the project root.
+
+To get a Clojurescript repl, run `boot cljsrepl`. In another tab, run `boot
+repl -c` to start a client-only repl. In that repl, run `(start-repl)`. Load
+the application in a browser (localhost:3000), and in the repl client you
+should see a "connected!" message after a moment, and now you should be able to
+require cljs files and send javascript to the browser.
```

Comment:
I have discovered that this has to be run in a normal browser window and an incognito window won't work, which I think we should call out in the README